hooks/useSheetRendererData.tsx

```diff
@@ -12,10 +12,24 @@ const useSheetRendererData = () => {
   const uploadPfp = async (pfpRef) => {
     if (!pfpRef?.current) return ""
     try {
-      const blob = await domtoimage.toBlob(pfpRef.current)
+      const options = {
+        quality: 1,
+        scale: 2,
+        style: {
+          transform: "scale(2)",
+          transformOrigin: "top left",
+          width: `${pfpRef.current.offsetWidth}px`,
+          height: `${pfpRef.current.offsetHeight}px`,
+        },
+        width: pfpRef.current.offsetWidth * 2,
+        height: pfpRef.current.offsetHeight * 2,
+      }
```

Comment:
are these options changing anything about the `legacy` images in firebase?

new images are already high quality right?

are you sure these changes are needed?